### DISCUSSION TITLE
Can this docker container run on armv6l?

### DISCUSSION THREAD

**Discussion by Itzamna44** [zwave-js/zwave-js-ui#664] [General]
Hello.
I'm trying to run this docker container on Raspberry Pi Model B Rev 2 
uname -a
Linux zwave 5.10.11+ #1399 Thu Jan 28 12:02:28 GMT 2021 armv6l GNU/Linux

But container keeps on restarting with error: Restarting (139) 3 seconds ago

Can this docker container run on armv6l?

**Comment by robertsLando** [comment]
It should AFAIK. You could try to clone the repo and manually build it with the Dockerfile otherwise
